src/LCT.PackageIdentifier/NugetProcessor.cs

```diff
@@ -344,11 +345,11 @@ private static string GetArtifactoryRepoName(List<AqlResult> aqlResultList, Comp
                 if (isTrue)
                 {
                     internalComponents.Add(currentIterationItem);
-                    isInternal.Value = "true";
+                    isInternal.Value = "TRUE";
```

Comment:
any reason why you made the value in CAPS?

Reply:
Back to normal